ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.14__periodics.yaml

```diff
@@ -151,6 +151,17 @@ tests:
       MCE_VERSION: "2.3"
       ODF_OPERATOR_SUB_CHANNEL: stable-4.13
     workflow: hypershift-mce-kubevirt-baremetalds-conformance
+- as: e2e-mce-power-conformance
+  cron: 0 4 * * *
+  steps:
+    cluster_profile: aws
```

Comment:
@mkumatag  Hi, I am curious about the use of an AWS cluster profile for IBM Cloud power resources. Could you please explain the reasoning behind it? thanks

Reply:
this is for hosting the HCP(management cluster) running in the x86 architecture and workers(nodepools) will be running in the IBM Cloud(PowerVS)